### PR TITLE
Feature/jng 2668 update UI design

### PR DESCRIPTION
**Pull request by fruska**

### CODE REVIEW

**Review by knobico — APPROVED** [review]

File: lib/src/judo_button.dart [hunk 1]
```diff
@@ -42,16 +42,42 @@ class JudoButton extends StatelessWidget {
...
-          : RaisedButton (
-            child: label != null ? Text(label) : Text(''),
-            onPressed: disabled ? null : onPressed,
-          ),
+      child: getButton(context),
     );
   }
+
+    Widget getButton(BuildContext context) {
+      return label != null && icon != null ?
```
Comment (noherczeg) [inline lib/src/judo_button.dart]: I'd prefer if-else statements. Multiple layers of ternary operators are harder to read.

File: lib/src/judo_combobox.dart [hunk 2]
```diff
@@ -60,17 +62,15 @@ class _JudoComboBoxState<T> extends State<JudoComboBox<T>> {
...
+        data: JudoComponentCustomizer.get().getInputComboboxThemeCustomizer(theme, widget.disabled, widget.readOnly, widget.inCard),
         child: Container(
           decoration: JudoComponentCustomizer.get().getInputBoxCustomizer(widget.disabled, widget.readOnly),
-          child: DropdownButtonFormField<T>(
-              decoration: JudoComponentCustomizer.get().getInputTextDecoration(widget.label, widget.icon, null)
-                  .copyWith(
-                labelText: widget.mandatory ? widget.label + ' *' : widget.label,
-              ),
+              child: DropdownButtonFormField<T>(
+              decoration: JudoComponentCustomizer.get().getInputComboboxDecoration(widget.label, widget.icon, null, widget.mandatory),
```
Comment (noherczeg) [inline lib/src/judo_combobox.dart]: Indentation is off, makes it harder to track what resides under what. If possible, can you please re-align it?

File: lib/src/judo_input_text.dart [hunk 3]
```diff
@@ -66,33 +68,29 @@ class JudoInputTextState extends State<JudoInputText> {
...
-              decoration: JudoComponentCustomizer.get().getInputBoxCustomizer(widget.disabled, widget.readOnly)
-             ),
-          data: JudoComponentCustomizer.get().getInputTextThemeCustomizer(theme, widget.disabled, widget.readOnly),
+      Theme(
+        child:
+        Container(
+            margin: widget.multiline ? EdgeInsets.symmetric(vertical: 10.0) : null,
+            child: TextField(
+              controller: controller,
+              readOnly: widget.disabled ? true : widget.readOnly,
```
Comment (noherczeg) [inline lib/src/judo_input_text.dart]: `readOnly: widget.disabled || widget.readOnly,` ?

File: lib/src/judo_input_text.dart [hunk 4]
```diff
@@ -66,33 +68,29 @@ class JudoInputTextState extends State<JudoInputText> {
...
+              enabled: widget.disabled ? false : !widget.readOnly,
```
Comment (noherczeg) [inline lib/src/judo_input_text.dart]: `enabled: !widget.disabled && !widget.readOnly,` ?

File: lib/src/judo_table.dart [hunk 5]
```diff
@@ -120,7 +127,8 @@ class JudoTable extends StatelessWidget {
                 if (states.contains(MaterialState.selected))
                   return Theme.of(context).colorScheme.primary.withOpacity(0.08);
                 // Even rows will have a grey color.
-                if (index % 2 == 0) return Colors.grey.withOpacity(0.05);
+                if (index % 2 == 0) return inCard ? Theme.of(context).backgroundColor : Theme.of(context).cardColor;
```
Comment (noherczeg) [inline lib/src/judo_table.dart]: I believe empty return in the end is never reached. Therefore this should do the same, but also would be a bit easier to read: 
```
if (index % 2 == 0) {
  return inCard ? Theme.of(context).backgroundColor : Theme.of(context).cardColor;
}
return inCard ? Theme.of(context).cardColor : Theme.of(context).backgroundColor;
```

**Review by noherczeg — CHANGES_REQUESTED** [review]

**Review by robertcsakany — APPROVED** [review]

**Review by noherczeg — APPROVED** [review]

**Pull request merged**